extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -99,7 +99,7 @@ function waitForEvent(env, name) {
   });
 }
 
-describe.configure().ifNewChrome().run('Bind', function() {
+describe.configure().ifIe().ifNewChrome().run('Bind', function() {
```

Comment:
Hmm. I did some more testing, and it turns out that due to how `ifFoo` fully replaces the platform on which tests are run, chaining together `ifIe` and `ifChrome`, for example, does not have the effect we desire. Chaining together `enableIE` and `ifChrome` doesn't work either. The only way to run a test on more than one platform is to skip all the others, or to create a whole set of `enableFoo` methods that can then be chained together. Taking another look to see if there's a less invasive way of making this work.